mailpoet/assets/js/src/email-editor/engine/store/types.ts

```diff
@@ -27,24 +27,31 @@ export type ExperimentalSettings = {
 export type EmailEditorSettings = EditorSettings & ExperimentalSettings;
 
 export type EmailStyles = {
-  layout: {
-    background: string;
+  spacing: {
+    blockGap: string;
     padding: {
       bottom: string;
       left: string;
       right: string;
       top: string;
     };
-    width: string;
   };
-  colors: {
-    background: string;
+  color: {
+    background: {
+      content: string;
+      layout: string;
+    };
+    text: string;
   };
   elements: {
     h1: {
-      color: string;
-      fontFamily: string;
-      fontWeight: string;
+      color: {
+        text: string;
+      };
+      typography: {
+        fontFamily: string;
+        fontWeight: string;
+      };
```

Comment:
The change was added earlier, but I struggle to understand why we define only h1 in this type definition. Shouldn't we have some common type for element styles?

Reply:
Oh I see now h1 is used in `email-editor/engine/components/styles-sidebar/preview.tsx`.
Ok. I guess we can keep it for now. We are aiming to align with theme.json, so I'm hoping in the future, we will be able to grab some types from Gutenberg.

Reply:
@costasovo I think we will need to add more definitions when we implement the typography panel. So, our theme.json will be more robust in the future.